lib/printer.js

```diff
@@ -1155,6 +1155,36 @@ function genericPrintNoParens(path, options, print) {
 
         return concat(parts);
 
+    case "TemplateElement":
+        return n.value.raw;
+
+    case "TemplateLiteral":
+        var expressions = [];
+        var parts = [];
+        var i = 0;
+
+        path.each(function(childPath) {
+            expressions.push(print(childPath));
+        }, "expressions")
+
+        path.each(function(childPath) {
```

Comment:
If that doesn't work, `childPath.getName()` should also be the index of the child in the `.expressions` list.